tests/e2e/cmux_test.go

```diff
@@ -63,10 +64,19 @@ func TestConnectionMultiplexing(t *testing.T) {
 	} {
 		t.Run(tc.name, func(t *testing.T) {
 			ctx := context.Background()
-			cfg := etcdProcessClusterConfig{clusterSize: 1, clientTLS: tc.serverTLS, enableV2: true, clientHttpSeparate: tc.separateHttpPort}
+			cfg := etcdProcessClusterConfig{
+				clusterSize:        1,
+				clientTLS:          tc.serverTLS,
+				enableV2:           true,
+				clientHttpSeparate: tc.separateHttpPort,
+				stopSignal:         syscall.SIGTERM, // check graceful stop
```

Comment:
In main branch, expect package uses SIGTERM by default. In 3.4, I uses `stopSignal` as option so that there is no big change in the whole e2e.